aws/resource_aws_appsync_graphql_api.go

```diff
@@ -360,3 +376,43 @@ func flattenAppsyncGraphqlApiUserPoolConfig(userPoolConfig *appsync.UserPoolConf
 
 	return []interface{}{m}
 }
+
+func resourceAwsAppsyncSchemaPut(apiId string, d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).appsyncconn
+
+	if d.HasChange("schema") {
+		input := &appsync.StartSchemaCreationInput{
+			ApiId:      aws.String(apiId),
+			Definition: ([]byte)(d.Get("schema").(string)),
+		}
+		if _, err := conn.StartSchemaCreation(input); err != nil {
+			return err
+		}
+
+		activeSchemaConfig := &resource.StateChangeConf{
+			Pending: []string{"PROCESSING"},
```

Comment:
Nit: We should use the [AWS Go SDK provided constants](https://docs.aws.amazon.com/sdk-for-go/api/service/appsync/#pkg-constants) here and below, e.g.

```suggestion
			Pending: []string{appsync.SchemaStatusProcessing},
```